### PR TITLE
Migrate options to settings for some plugins

### PR DESCRIPTION
**Pull request by LordEliasTM**
migrated:
- callTimer
- messageClickActions
- messageLogger
- platformIndicators

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -12,5 +12,8 @@
     "javascript.format.semicolons": "insert",
     "typescript.format.semicolons": "insert",
     "typescript.preferences.quoteStyle": "double",
-    "javascript.preferences.quoteStyle": "double"
-}
+    "javascript.preferences.quoteStyle": "double",
+    "[css]": {
+        "editor.formatOnSave": false
```
Comment (Vendicated) [inline .vscode/settings.json:17]: ?

**Review by LordEliasTM — COMMENTED** [review]
reverted the settings.json and removed that old settings part

**Comment by Vendicated** [comment]
thanks

**Comment by LordEliasTM** [comment]
You're welcome

**Pull request merged**